src/codewhisperer/vue/backend.ts

```diff
@@ -0,0 +1,153 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import * as nls from 'vscode-nls'
+const localize = nls.loadMessageBundle()
+import * as os from 'os'
+import * as vscode from 'vscode'
+import * as fs from 'fs'
+import * as path from 'path'
+import { VueWebview } from '../../webviews/main'
+import { isCloud9 } from '../../shared/extensionUtilities'
+import globals from '../../shared/extensionGlobals'
+import { telemetry, CodewhispererLanguage, CodewhispererGettingStartedTask } from '../../shared/telemetry/telemetry'
+
+export class CodeWhispererWebview extends VueWebview {
+    public readonly id = 'CodeWhispererWebview'
+    public readonly source = 'src/codewhisperer/vue/index.js'
+
+    public constructor(private readonly start: string) {
+        super()
+    }
+    //This function is called when the extension is activated to check whether is it the first time the user is using the extension or not
+    public async showAtStartUp(): Promise<string | void> {
+        return this.start
+    }
+
+    private isFileSaved: boolean = false
+    private getLocalFilePath(fileName: string): string {
+        //This will store the files in the global storage path of VSCode
+        return path.join(globals.context.globalStoragePath, fileName)
```

Comment:
`globalStoragePath` is deprecated, use `globalStorageUri` instead. Also since the `globalStorageUri` folder is globally used you should created a subfolder `return path.join(globals.context.globalStorageUri, 'codeWhispererStartup', fileName)` so all of this code is contained to a folder.

Reply:
Tried using `globalStorageUri` 
`  return path.join(globals.context.globalStorageUri.toString(), fileName)`
 but I got this error which shows not able to open 
![image](https://github.com/aws/aws-toolkit-vscode/assets/141967714/01e00dab-2c9d-418b-a566-67ffde6f0af0)

So, Opted to go with globalStorage.

Reply:
path.join() won't work on a URI. you would need to do something like

    path.join(globals.context.globalStorageUri.fsPath, fileName)

Reply:
Updated !